### PR TITLE
Feat/food grade certificate

### PR DESCRIPTION
**Pull request by rhofvendahl**

### CODE REVIEW

File: docs/openapi/components/schemas/credentials/FoodGradeInspectionCertificate.yml [hunk 1]
```diff
@@ -0,0 +1,484 @@
+$linkedData:
+  term: FoodGradeInspectionCertificate
+  '@id': https://w3id.org/traceability#FoodGradeInspectionCertificate
+title: Food Grade Inspection Certificate
+description: >-
+  The USDA grade shields, official seals and labels are reputable symbols of the quality and
```
Comment (TallTed) [inline docs/openapi/components/schemas/credentials/FoodGradeInspectionCertificate.yml:6]:
```suggestion
  The USDA grade shields, official seals, and labels are reputable symbols of the quality and
```

File: docs/openapi/components/schemas/credentials/FoodGradeInspectionCertificate.yml [hunk 2]
```diff
@@ -0,0 +1,484 @@
...
+  integrity of American agricultural products.  Large-volume buyers such as grocery stores,
+  military institutions, restaurants, and even foreign governments use the quality grades as a
+  common “language," making business transactions easier.  
+
+  The USDA shields and labels assure consumers that the products they buy have gone through a
+  rigorous review process by highly-skilled graders & auditors that follow the official grade
+  standards and process standards developed, maintained and interpreted by USDA’s Agricultural
```
Comment (TallTed) [inline docs/openapi/components/schemas/credentials/FoodGradeInspectionCertificate.yml:13]:
```suggestion
  standards and process standards developed, maintained, and interpreted by USDA’s Agricultural
```

File: docs/openapi/components/schemas/credentials/FoodGradeInspectionCertificate.yml [hunk 3]
```diff
@@ -0,0 +1,484 @@
...
+        "address": {
+          "type": [
+            "PostalAddress"
+          ],
+          "organizationName": "Ace Foodstuffs",
+          "streetAddress": "853 Wisozk River",
+          "addressLocality": "New Noemyfort",
+          "addressRegion": "New Mexico",
+          "postalCode": "18047-2038",
+          "addressCountry": "Togo"
```
Comment (TallTed) [inline docs/openapi/components/schemas/credentials/FoodGradeInspectionCertificate.yml:116]: New Mexico is in the USA, and that's a US ZIP+4 Code... 
```suggestion
          "addressRegion": "New Mexico",
          "postalCode": "18047-2038",
          "addressCountry": "USA"
```

File: docs/openapi/components/schemas/credentials/FoodGradeInspectionCertificate.yml [hunk 4]
```diff
@@ -0,0 +1,484 @@
...
+          "type": "Organization",
+          "name": "Green Fields",
+          "description": "Growing & packaging for high quality produce",
+          "email": "sales@greenfields.org",
+          "phoneNumber": "+1-555-865-8495"
+        },
+        "purchaser": {
+          "type": "Organization",
+          "name": "Ace Foodstuffs",
+          "description": "Ag goods shipping & distribution",
```
Comment (TallTed) [inline docs/openapi/components/schemas/credentials/FoodGradeInspectionCertificate.yml:294]:
```suggestion
          "description": "Agricultural goods shipping & distribution",
```

**Review by TallTed — CHANGES_REQUESTED** [review]

File: docs/openapi/components/schemas/credentials/FoodGradeInspectionCertificate.yml (hunk 3 above)
Comment (rhofvendahl) [reply docs/openapi/components/schemas/credentials/FoodGradeInspectionCertificate.yml:116]: Sure - these are generated automatically but happy to change it

Comment (TallTed) [reply docs/openapi/components/schemas/credentials/FoodGradeInspectionCertificate.yml]: @rhofvendahl -- Ah, then whatever's generating these sample values needs some attention. We do want fake addresses, etc., for sample data (so we run no risk of causing trouble for real entities), but it should be usable with, for instance, tools that display such entities on real maps, tools that calculate distances between them, etc., for evaluation of both those tools and the traceability vocab. /cc @mprorock @OR13

**Pull request merged**